src/SFA.DAS.EmployerFinance/Services/PaymentService.cs

```diff
@@ -181,10 +185,12 @@ private Task<Models.ApprenticeshipProvider.Provider> GetProvider(int ukPrn)
 
                     if (providerView == null)
                     {
-                        providerView = _apprenticeshipInfoService.GetProvider(ukPrn);
+                        //providerView = _apprenticeshipInfoService.GetProvider(ukPrn);
```

Comment:
This never calls for the provider name other than looking into the cache for it so we get a 

if (providerView == null) 

Then directly after

if (providerView != null) 


No actual call to get it. It may be that this method is now deprecated and should be re-moved altogether? Some work needs to doing to either delete or make it do something.

Reply:
Commented out line in investigation, forgot to remove comment. Will fix